plugins/Live/Live.php

```diff
@@ -48,10 +51,77 @@ public function addJsGlobalVariables(&$out)
     {
         $actionsToDisplayCollapsed = (int)StaticContainer::get('Live.pageViewActionsToDisplayCollapsed');
         $out .= "
+        piwik.visitorLogEnabled = ".json_encode(self::isVisitorLogEnabled()).";
+        piwik.visitorProfileEnabled = ".json_encode(self::isVisitorProfileEnabled()).";
         piwik.visitorLogActionsToDisplayCollapsed = $actionsToDisplayCollapsed;
         ";
     }
 
+    public static function isVisitorLogEnabled($idSite = null)
+    {
+        [$profileEnabled, $logEnabled] = self::getSettings($idSite);
+
+        return $logEnabled;
+    }
+
+    public static function isVisitorProfileEnabled($idSite = null)
+    {
+        [$profileEnabled, $logEnabled] = self::getSettings($idSite);
+
+        return $profileEnabled;
+    }
+
+    private static function getSettings($idSite = null)
+    {
+        if (empty($idSite)) {
+            $idSite = Common::getRequestVar('idSite', 0, 'int');
+        }
+
+        $cache = Cache::getTransientCache();
```

Comment:
fyi @sgiehl in future we probably wouldn't need to cache these settings. If so, we could add a generic transient cache here to all settings: https://github.com/matomo-org/matomo/blob/4.0.0-b3/core/Settings/Storage/Factory.php#L134 but they shouldn't be majorly slow and partially already transient cached in the backend potentially